implementation/src/test/java/io/smallrye/metrics/exporters/JsonExporterTest.java

```diff
@@ -0,0 +1,40 @@
+package io.smallrye.metrics.exporters;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.eclipse.microprofile.metrics.Meter;
+import org.eclipse.microprofile.metrics.MetricRegistry;
+import org.junit.Test;
+
+import io.smallrye.metrics.MetricRegistries;
+import io.smallrye.metrics.app.MeterImpl;
+
+public class JsonExporterTest {
+
+    @Test
+    public void exportOneMetric_metered() {
```

Comment:
This should have a more descriptive name, given that the purpose of this test is to verify that the exporter can handle different metric impl classes... `testExportOfDifferentMeterImplementations` or something like that?